resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DeletedOrErroredKubernetesClusterChecker.scala

```diff
@@ -47,7 +47,8 @@ object DeletedOrErroredKubernetesClusterChecker {
                 }
               }
             } yield clusterOpt.fold(none[KubernetesCluster])(_ => Some(cluster))
-          case CloudContext.Azure(_) => logger.warn("Azure is not supported yet").as(none)
+          case CloudContext.Azure(_) =>
+            logger.warn("Resource validator not supported for Azure k8s clusters").as(none)
```

Comment:
As mentioned above I don't think `resourceValidator` ever really makes sense for our AKS implementation. The reason is that Leo doesn't own the AKS/nodepool provisioning -- it's all part of the landing zone.